docs/leap_sapi/sapi_rest.rst

```diff
@@ -812,15 +865,16 @@ sampler,see the :ref:`sapi_rest_full_examples` section.
     >>> r.json()   # doctest: +SKIP
```

Comment:
A couple of things are unclear to me here: 

1. "example 201 responses" --> but I'm seeing 200, not 201.
```
>>> r
<Response [200]>
```
2. I'm seeing the JSON response wrapped as a list:
```
>>> r.json()
[{'status': 'PENDING',
  'id': '306f60a ...
sion 2'}]
```